tests/Base.php

```diff
@@ -96,7 +96,9 @@ protected function _createIndex(?string $name = null, bool $delete = true, int $
         $client = $this->_getClient();
         $index = $client->getIndex($name);
 
-        $index->create(['settings' => ['index' => ['number_of_shards' => $shards, 'number_of_replicas' => 1]]], $delete);
+        $index->create(['settings' => ['index' => ['number_of_shards' => $shards, 'number_of_replicas' => 1]]], [
```

Comment:
Out of curiosity: The old one would still work but we discourage using it like this?

Reply:
Yes the old one will still work, but it will be deprecated so this is to avoid triggering the future deprecation.